modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/CommonAnalysisPlugin.java

```diff
@@ -588,54 +505,17 @@ public List<PreConfiguredTokenizer> getPreConfiguredTokenizers() {
         tokenizers.add(PreConfiguredTokenizer.singleton("letter", LetterTokenizer::new));
         tokenizers.add(PreConfiguredTokenizer.singleton("whitespace", WhitespaceTokenizer::new));
         tokenizers.add(PreConfiguredTokenizer.singleton("ngram", NGramTokenizer::new));
-        tokenizers.add(PreConfiguredTokenizer.indexVersion("edge_ngram", (version) -> {
-            if (version.onOrAfter(IndexVersions.V_7_3_0)) {
-                return new EdgeNGramTokenizer(NGramTokenizer.DEFAULT_MIN_NGRAM_SIZE, NGramTokenizer.DEFAULT_MAX_NGRAM_SIZE);
-            }
-            return new EdgeNGramTokenizer(EdgeNGramTokenizer.DEFAULT_MIN_GRAM_SIZE, EdgeNGramTokenizer.DEFAULT_MAX_GRAM_SIZE);
```

Comment:
@javanna ,
Since you removed 7.x related/deprecated codes from getPreConfiguredTokenizers() method, DEFAULT_MAX_NGRAM_SIZE of a `PreConfiguredTokenizer` for "**edge_ngram**" has changed from 1 to 2.

Is there any dependency problem? I hope not :) 
I agree with that it's practical to use `DEFAULT_MAX_GRAM_SIZE `of 2 not 1 as you merged.

(releated PR : https://github.com/apache/lucene/pull/13814)